### ISSUE TITLE
[Bug]:  Missing information And invalid value on Solax Boost X1 

### ISSUE THREAD

**Issue by ioniks** [wills106/homeassistant-solax-modbus#817]
### Describe the bug

There is invalid information and missing information :
Grid Export (W) = Measured Power = nearly Pw Power total
Grid Import (W) = always 0
Total Grid Export = Total Grid Import = Inconnu (in french)
Mesured Power L1 to L3 not needed on X1
Inverter Power : "Inconnu"
Missing entities : Today Export energy, Today Import energy, House load

### Integration Version

2024.03.6

### Homeassistant core version

2024.3.3

### Inverter brand

Solax Power

### Plugin used

plugin_solax.py

### Serial prefix

 XB333

### Inverter firmware versions

ARM: 147, DSP : 229

### Connection Method

Solax powket Wifi 3.0

### Dongle firmware

Can't check it at this time

### Detailed Error Log

No error

**Comment by wills106** [comment]
I can correct Measured power. But there aren't any additional register to add.
There hasn't been an update to the Modbus docs since 2021 for this Inverter.
I have already tried to reach out to SolaX to see if there has been an update to the documentation https://github.com/wills106/homeassistant-solax-modbus/issues/743#issuecomment-1970613253

**Comment by wills106** [comment]
I have corrected Measured Power's in [2024.03.7](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2024.03.7)

**Comment by ioniks** [comment]
@wills106 hi, i have received a doc from solax

[SolaxPower Single Phase External Communication Protocol - X1 Series V3.0_20240326165841.pdf](https://github.com/wills106/homeassistant-solax-modbus/files/14755591/SolaxPower.Single.Phase.External.Communication.Protocol.-.X1.Series.V3.0_20240326165841.pdf)

**Comment by wills106** [comment]
That's correct, it's a Non standard protocol https://github.com/wills106/homeassistant-solax-modbus/issues/743#issuecomment-1963746436

**Comment by ioniks** [comment]
I have a response from solax no other documentation.

I can help test or retrieve modbus info to find them? If so how can I help?
